ppsci/data/dataset/era5climate_dataset.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2025 PaddlePaddle Authors. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import numbers
+import os
+import random
+import time
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import h5py
+import numpy as np
+import paddle
+import xarray as xr
```

Comment:
xarray不是PaddleScience的依赖，请修改

```suggestion
try:
    import xarray as xr
except ModuleNotFoundError:
    pass
```